src/Interfaces/IParticle.ts

```diff
@@ -15,6 +15,7 @@ import type { Emitter } from "../Classes/Emitter";
 
 export interface IParticle {
 	randomIndexData?: number;
+	links: IParticle[];
```

Comment:
this property shouldn't be here, this interface will be exposed to custom shapes and presets. If you need this property use `Particle` class as a parameter instead of `IParticle` interface.

Reply:
Are we not allowing line linking on custom shapes? The purpose of that array is the store the line link relationships

Reply:
shape drawers should only draw the current particle shape, not modify anything about it, linked lines are drawed only by the library